package.json

```diff
@@ -139,7 +140,7 @@
     "configcat-js": "5.4.0",
     "configcat-node": "6.7.0",
     "contentful": "7.14.6",
-    "contentful-management": "6.0.5",
+    "contentful-management": "10.12.0",
```

Comment:
Need to make sure the translation script still works. Try this:

```
yarn nx extract-strings shared-translations
```

Reply:
Seems to work 
![image](https://user-images.githubusercontent.com/43557895/187655039-6a80ab36-8a00-4391-9f11-59ff13f45af5.png)